packages/mdc-menu-surface/_mixins.scss

```diff
@@ -20,17 +20,130 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/theme/mixins";
 @import "@material/shape/mixins";
+@import "@material/animation/variables";
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "./variables";
 
-@mixin mdc-menu-surface-ink-color($color) {
-  @include mdc-theme-prop(color, $color);
+//
+// Public
+//
+
+@mixin mdc-menu-surface($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  // postcss-bem-linter: define menu-surface
+  .mdc-menu-surface {
+    @include mdc-menu-surface-base_($query);
+    @include mdc-elevation($z-value: 8, $query: $query);
+
+    @include mdc-menu-surface-fill-color(surface, $query);
+    @include mdc-menu-surface-ink-color(on-surface, $query);
+    @include mdc-menu-surface-shape-radius(medium, $query);
+
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-rtl-reflexive-property(transform-origin, top left, top right);
+    }
+  }
+
+  @include mdc-feature-targets($feat-structure) {
```

Comment:
Invert these to put the feature-targets within each selector. Same for within `base_`.